src/main/java/io/quarkus/registry/app/maven/cache/MavenCacheClearInterceptor.java

```diff
@@ -0,0 +1,28 @@
+package io.quarkus.registry.app.maven.cache;
+
+import javax.inject.Inject;
+import javax.interceptor.AroundInvoke;
+import javax.interceptor.Interceptor;
+import javax.interceptor.InvocationContext;
+
+/**
+ * Interceptor that fire a event to notify that the cache should be cleared
+ * @author Phillip Kruger (phillip.kruger@redhat.com)
```

Comment:
It would be nice to follow the Quarkus coding standards and avoid `@author` tags

Reply:
Sure, sorry (my IDE add them)